packages/eslint-plugin/src/constants.ts

```diff
@@ -1,2 +1,5 @@
 export const CLASS_FIELDS = ['class', 'classname']
 export const AST_NODES_WITH_QUOTES = ['Literal', 'VLiteral']
+
+// supports the default attributify prefix 'un-', and includes other potential UnoCSS-related prefixes for broader compatibility
+export const ATTRIBUTIFY_PREFIXES = ['un-', 'uno-', 'unocss-', 'un:', 'uno:', 'unocss:']
```

Comment:
For attribute mode, we only have the default prefix `un-`
I don't know if this prefix list can be passed in via configuration.

Reply:
I haven’t looked deeply into the code yet, but as far as I know right now, we don’t have access to the user’s configuration, which makes the prefixes not configurable at the moment.
I'll explore it more when I'm free.

Reply:
Yeah, I would feel this should be configurable, and the default to only be `un-` to match the preset's default